src/main/java/it/gov/pagopa/reward/notification/service/exports/ForceOrganizationExportServiceImpl.java

```diff
@@ -0,0 +1,42 @@
+package it.gov.pagopa.reward.notification.service.exports;
+
+import it.gov.pagopa.reward.notification.model.RewardOrganizationExport;
+import it.gov.pagopa.reward.notification.repository.RewardOrganizationExportsRepository;
+import it.gov.pagopa.reward.notification.service.csv.out.ExportRewardNotificationCsvService;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
+
+import java.time.LocalDate;
+import java.util.List;
+
+@Service
+@Slf4j
+public class ForceOrganizationExportServiceImpl implements ForceOrganizationExportService {
+
+    private final RewardOrganizationExportsRepository rewardOrganizationExportsRepository;
+    private final ExportRewardNotificationCsvService exportRewardNotificationCsvService;
+
+
+    public ForceOrganizationExportServiceImpl(RewardOrganizationExportsRepository rewardOrganizationExportsRepository, ExportRewardNotificationCsvService exportRewardNotificationCsvService) {
+        this.rewardOrganizationExportsRepository = rewardOrganizationExportsRepository;
+        this.exportRewardNotificationCsvService = exportRewardNotificationCsvService;
+    }
+
+    @Override
+    public Flux<List<RewardOrganizationExport>> execute(LocalDate notificationDateToSearch) {
+        LocalDate now = LocalDate.now();
+
+        return rewardOrganizationExportsRepository.findByExportDate(now)
+                .flatMap(this::cleanTodayExport)
+                .thenMany(exportRewardNotificationCsvService.execute(notificationDateToSearch));
+    }
+
+    private Mono<RewardOrganizationExport> cleanTodayExport(RewardOrganizationExport x) {
```

Comment:
let's use a more anonymous name for this parameter, "r" for example is less anonymous than "x" :D

We could accept "x" in short lambdas, where is immediate its reading...
Inside a method with body it's better to avoid such names